emsdk_manifest.json

```diff
@@ -350,13 +388,13 @@
   {
     "version": "main",
     "bitness": 64,
-    "uses": ["python-3.9.2-nuget-64bit", "llvm-git-main-64bit", "node-20.18.0-64bit", "emscripten-main-64bit", "binaryen-main-64bit"],
```

Comment:
I think you can also delete the python-3.9.2 entries in this file

Reply:
If we did that, then people cannot manually choose to install the previous python 3.9.2 to e.g. compare if updating python to 3.13.3 was the root cause of a regression.

i.e. with retaining that, users can still install combinations of the individual tools that they like.